apps/src/applab/applab.js

```diff
@@ -894,10 +899,16 @@ Applab.init = function(config) {
     isEmbedView: !!config.embed,
     isReadOnlyWorkspace: !!config.readonlyWorkspace,
     isShareView: !!config.share,
-    isViewDataButtonHidden: !!config.level.hideViewDataButton
+    isViewDataButtonHidden: !!config.level.hideViewDataButton,
+    instructionsMarkdown: studioApp.getMarkdownInstructions(config.level)
   }));
 
-  Applab.reduxStore.dispatch(changeInterfaceMode(Applab.startInDesignMode() ? ApplabInterfaceMode.DESIGN : ApplabInterfaceMode.CODE));
+  // TODO - danger here (and with other config), is we now have the potential
+  // for two opinions - one on the config object, and another in the redux store
+  Applab.reduxStore.dispatch(setInstructionsInTopPane(config.showInstructionsInTopPane));
```

Comment:
You only set this once, as if it's a constant for the session.  If so, why didn't you roll this into the setInitialLevelProps action?